backend/app/store/service/service.go

```diff
@@ -996,3 +1049,13 @@ func (s *DataStore) getSecret(siteID string) (secret string, err error) {
 	}
 	return secret, nil
 }
+
+func (s *DataStore) indexComment(comment store.Comment) error {
+	if s.SearchService != nil {
+		err := s.SearchService.Index(comment)
+		if err != nil {
+			return err
```

Comment:
I'm not sure if it makes sense (have not checked what the error is), but maybe we want to wrap the error to indicate what comment failed to index?